spec/views/post_show_spec.rb

```diff
@@ -49,8 +40,9 @@
     visit "/user/#{user.id}/post/#{post.id}"
   end
 
-  it 'displays post author name' do
-    expect(page).to have_content(user.name)
+  it "I can see the user's name." do
+    visit '/user/1/post'
+    page.has_content?(user.name)
   end
 
   it 'counts total number of comments' do
```

Comment:
- [ ] Let's ensure to include test for the below scenarios for _post show_ views: 
   * I can see the post's title
   * I can see who wrote the post
   * I can see how many likes it has
   * I can see the post body
   * I can see the username of each commentor
   * I can see the comment each commentor left